docs/source/troubleshooting/troubleshooting-misc.adoc

```diff
@@ -93,3 +93,29 @@ Workaround: Delete the existing {project} VM and start again.
 $ minishift delete --force
 $ minishift start
 ----
+
+[[Remove-password-from-keychain]]
+== Removing registration password from OS native keychain
+
```

Comment:
I'd suggest specifying that this only applies to the RHEL ISO. (Unless I'm misunderstanding something, these instructions only apply for RHEL.)

Also, when would we expect users to do this? If they've changed their password? Are there any other situations where this would be useful to know? I'm asking so that we can mention those situations here -- this could potentially help users who search the documentation for their particular problem.

Reply:
> I'd suggest specifying that this only applies to the RHEL ISO

Yes, this only applies to RHEL iso

> when would we expect users to do this? If they've changed their password?

Only when they have changed the password, or mistakenly entered a wrong password which got stored in the keychain